cloud/docker/docker.py

```diff
@@ -713,6 +721,8 @@ def __init__(self, module):
             self.links = self.get_links(self.module.params.get('links'))
 
         self.env = self.module.params.get('env', None)
+        self.env_file = self.module.params.get('env_file', None)
```

Comment:
We might discuss about removing the `env` and `env_file` properties from `self`. As far as I can see they won't be needed after `self.environment` has been created as a merged result of both params. WDYT?